server/routers/viewer.tsx

```diff
@@ -383,6 +384,31 @@ const loggedInViewerRouter = createProtectedRouter()
       };
     },
   })
+  .query("availability", {
+    async resolve({ ctx }) {
+      const { prisma, user } = ctx;
+      const availabilityQuery = await prisma.availability.findMany({
+        where: {
+          userId: user.id,
+        },
+      });
+      const schedule = availabilityQuery.reduce(
+        (schedule: Schedule, availability) => {
+          availability.days.forEach((day) => {
+            schedule[day].push({
+              start: new Date(new Date().toDateString() + " " + availability.startTime.toTimeString()),
+              end: new Date(new Date().toDateString() + " " + availability.endTime.toTimeString()),
+            });
```

Comment:
maybe this shouldn't actually pass back a date and rather just pass the time as-is from the db?

the type could be typed like 

```ts
type Time = `${number}${number}:${number}${number}:${number}${number}`;
```